lua/dap.lua

```diff
@@ -333,6 +410,9 @@ function Session:event_stopped(stopped)
     return
   end
   self.stopped_thread_id = stopped.threadId
+  if stopped.reason == 'exception' then
+    self:_show_exception_info()
+  end
```

Comment:
Do you think we should chain the `threads`  request call below after the `exceptionInfo` response has been processed? 

I think otherwise the way the repl buffer is mutated may not have a fixed order.

Reply:
:+1: It has to be the other way around. `exceptionInfo` needs to know `threadId`.